src/libraries/System.Configuration.ConfigurationManager/src/System/Configuration/SettingsSerializeAs.cs

```diff
@@ -1,12 +1,15 @@
 // Licensed to the .NET Foundation under one or more agreements.
 // The .NET Foundation licenses this file to you under the MIT license.
 
+using System;
+
 namespace System.Configuration
 {
     public enum SettingsSerializeAs
     {
         String = 0,
         Xml = 1,
+        [Obsolete(Obsoletions.BinaryFormatterMessage + @". Consider using Xml instead.", false)]
```

Comment:
This is a change to public surface area. I would have expected the ref to also be updated with this.